### PR TITLE
Admin delete service

### PR DESCRIPTION
**Pull request by at-trungtranqn**
Admin delete service

### CODE REVIEW

**Comment by ghost** [comment]
FIx confliict

**Comment by ghost** [comment]
LGTM !

**Comment by ghost** [comment]
fix conflict again =))

File: app/Http/Controllers/Admin/ServiceController.php [hunk 1]
```diff
@@ -25,6 +25,23 @@ public function index()
...
+     * @return void
+     */
+    public function destroy($id)
+    {
+        $service = Service::findOrFail($id);
+        if ($service->delete()) {
+            flash(__('Delete success'))->success();
+        } else {
+            flash(__('Delete failure'))->error();
+        }
```
Comment (ghost) [inline app/Http/Controllers/Admin/ServiceController.php:41]: Please redirect to index after process

Comment (at-trungtranqn) [reply app/Http/Controllers/Admin/ServiceController.php:41]: e fix rồi nha a :)

**Comment by ghost** [comment]
LGTM

**Pull request merged**